Spring-Boot/Spring-Boot-Student/src/test/java/tw/waterball/judgegirl/springboot/student/controllers/StudentControllerTest.java

```diff
@@ -290,6 +292,23 @@ void Given10StudentsAnd4Admins0_1_2_3_WhenGetAdminsWithSkip2Size2_ShouldRespondA
         assertEquals("admin3", students.get(1).name);
     }
 
+    @Test
+    void GivenStudentSignedUp_WhenDeleteTheStudent_ShouldSucceed() throws Exception {
+        StudentView student = signUpAndGetResponseBody(this.student);
+
+        deleteStudent(student.id);
+
+        assertTrue(studentRepository.findStudentById(student.id).isEmpty());
+    }
+
+    @Test
+    void WhenDeleteTheStudent_ShouldSucceed() throws Exception {
+        int studentId = 123;
+        deleteStudent(studentId);
+
+        assertTrue(studentRepository.findStudentById(studentId).isEmpty());
```

Comment:
The Given-When-Then statement in this testcase doesn't cover the testing of "a non-existing student should not be found in the repository", so we shouldn't test it here. 
Please remove this line.